e2e/cypress/integration/system_console/main_menu_spec.js

```diff
@@ -59,6 +57,9 @@ describe('Main menu', () => {
     });
 
     it('MM-T913 About opens About modal', () => {
+        // * Check if server has license
+        cy.apiRequireLicense();
```

Comment:
I realized that my thoughts in our chat which was "add inside the `it` block if wanted to maintain test grouping" is not accurate. I think the correct way is to move test that requires license in a separate file and then add metadata of `@enterprise` so that we'll be able to filter out those tests whenever we'll run Cypress in TE.
But I'm good as is. It's only something we should aim for so it's easier to implement TE and EE tests in the future.